### ISSUE TITLE
Web app prints "Invalid ESR" if `0x` prefix is missing

### ISSUE THREAD

**Issue by thass0** [google/aarch64-esr-decoder#85]

**Comment by qwandor** [comment]
If you omit the `0x` prefix then the number is interpreted as decimal. That is intentional, as sometimes it gets dumped that way in some logs or debuggers, but adding a comment explaining the behaviour seems like a good idea.

**Comment by thass0** [comment]
Yes, I think added a more detailed error message [here](https://github.com/google/aarch64-esr-decoder/blob/34da97e42737a638925ec87a0d62a38360a725de/aarch64-esr-web/src/lib.rs#L34) (and in the other cases where parsing fails) would really improve the user experience.

**Comment by qwandor** [comment]
@thass0 Please take a look at #86.

**Comment by thass0** [comment]
That looks great, thanks for the fix :grinning:
